Dmin_autocalculator_config.ini

```diff
@@ -0,0 +1,54 @@
+#works for both, Dmin_autocalculator and batchAnalysisStatistics. Make sure you set min_detectable_D in DIFFUSION_TYPE_PARAM to the value you get from the Dmin_autocalculator
+[SOFTWARE]
+# choose software, either ThunderSTORM or rapidSTORM
+software = ThunderSTORM
+
+[INPUT_DIRS]
+# path to the coverslips
+dir1 = D:\exampledir
+
+
+[MASK_WORDS]
+#words to filter in your directory, e.g. BACKGROUND
+mask = background
+
+[BACKGROUND]
+#whether to use background correction. If true give directories.
+background = false
+
+[BACKGROUND_DIRS]
+#directories to the background, make sure  they are in the same order as the INPUT_DIRS. Only necessary when background is set to true.
```

Comment:
"# directories to the background, make sure they are in the same order as the INPUT_DIRS. Only necessary when background is set to true."

(remove double space between sure and they)